### ISSUE TITLE
Restore filter button

### ISSUE THREAD

**Issue by rkorne** [dbeaver/dbeaver#19199]
After issue #18671 has been closed, the old functionality still isn't back. Instead, the little triangle in the header column duplicates both the sort function and the 'Filter' context menu.
The old filter button only had the 'Filter by value' function, which I would like to have back (as in versions up to 22.3.0)
Ordering is available both in the ordering attribute button and in the context menu, the filter button should be a shortcut to the 'Filter by value' function, more complex filtering can be done in the context menu.

**Comment by E1izabeth** [comment]
You can return back ordering arrows here by checking `Show attribute ordering`
![image](https://user-images.githubusercontent.com/28875055/221589756-3e57a8e1-8557-460d-a3ed-c17b69d9a021.png)

**Comment by E1izabeth** [comment]
Also we have a shortcut for `Filter by value` command.

**Comment by rkorne** [comment]
E1izabeth, thanks for the quick response. The 'return back ordering arrows' functions is nice, but not what I want. I'm aware of the Ctrl-F11 shortcut to the 'Filter by value' function, but I liked the old way with the button, not using the keyboard.

**Comment by Destrolaric** [comment]
@rkorne Hi, the filter button also can be enabled, via the following setting:
![image](https://user-images.githubusercontent.com/20579256/221911333-30e6f67d-6417-479d-84c3-0483fb5141e4.png)

**Comment by rkorne** [comment]
@Destrolaric Yes, but this duplicates ordering and filter context menu and requires an additional click to get to 'Filter by value'.
Up to 22.3.0 the filter button directly showed the 'Filter by value' dialog.

**Comment by vatsake** [comment]
I agree with @rkorne. That extra click is pointless.
The previous sort/order and filter by value buttons were all we ever needed.

Please add an option to reverse back to it.

**Comment by tianlongJ** [comment]
The sorting and filtering of the old version are more usable, but the current one requires two more clicks. I hope it can be restored.
